source/pt-BR/privacy-policy.blade.php

```diff
@@ -0,0 +1 @@
+@include('_pages.privacy-policy');
```

Comment:
Is this file necessary?
Would be best get the text to translate by other way.

Reply:
Not necessary, I was just test the style. It's not definitive.